types/reactstrap/lib/Row.d.ts

```diff
@@ -1,8 +1,8 @@
-interface Props {
+interface Props extends React.HTMLProps<any> {
```

Comment:
BTW: This is sadly not consistent throughout the whole defs and should be fixed for all components anyhow. @pollen8 I would appreciate your contribution! ;-)